Classes/Private/Manager/OCTManager.m

```diff
@@ -243,6 +266,21 @@ - (void)saveTox
     }
 }
 
+#pragma mark - OCTToxDelegate
+
+- (void)toxDidIterate:(OCTTox *)tox
+{
+    [self.files sendProgressNotificationsNow];
+}
+
+- (void)tox:(OCTTox *)tox friendConnectionStatusChanged:(OCTToxConnectionStatus)status friendNumber:(OCTToxFriendNumber)friendNumber
+{
+    // the rightful owner of this callback
+    [self.friends tox:tox friendConnectionStatusChanged:status friendNumber:friendNumber];
+    // needed for file resuming machinery
+    [self.files tox:tox friendConnectionStatusChanged:status friendNumber:friendNumber];
+}
```

Comment:
Maybe instead of breaking "forwarding" logic it would be better to implement some broadcast system that will allow submanagers to communicate? Any submanager will be able to broadcast some message and all other submanagers would be able to subscribe to it. So the logic would be
- submanager receives some event (friend connection status in our case)
- it send broadcast message to some broadcast center for all submanagers
- any interested submanager can subscribe to it (files submanager in our case).

However it may not be part of this issue, so we can leave it for now. However if you are interested you can implement it.